packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  CancelResponse,
+  ContinueResponse,
+  FunctionInfo,
+  ParametersGatherer
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as vscode from 'vscode';
+import { nls } from '../../messages';
+import {
+  CompositeParametersGatherer,
+  SfdxCommandlet,
+  SfdxWorkspaceChecker
+} from '../util';
+import { BaseTemplateCommand } from './baseTemplateCommand';
+import { FUNCTION_TYPE } from './metadataTypeConstants';
+
+export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
```

Comment:
Are we going to deal with the plugin not being installed in this PR or that will be on a different one ?

Reply:
I was considering this story as a happy path scenario. The pre-checks and post function creation tasks will be the next story.

Reply:
sounds good 👍